app/patients/reports/controller.js

```diff
@@ -838,14 +838,28 @@ export default AbstractReportController.extend(PatientDiagnosis, PatientVisits,
     let reportType = this.get('reportType');
     let startDate = this.get('startDate');
     if (reportType === 'status') {
+      if (isEmpty(endDate)) {
+        let now = new Date();
+        this.set('endDate', now);
+        endDate = this.get('endDate');
+      }
```

Comment:
Is this added code necessary?

Reply:
No, I don't think so. I was mistaken thinking status reports needed an end date. I've fixed it.

Reply:
thanks, I noticed something else too.

In patient, try selecting Admissions Detail, then set both dates to some date range, then go back to Patient Status and generate. As long as the properties are already set to something, it will still show a date range in the report title. Or if only one of the dates is set, that one will appear. Really, I am thinking we want no dates to ever appear in Patient Status title.

The two Inventory reports that don't have start date, don't seem to have this problem. You can set both dates on one report, then switch to Inventory Valuation but it will still only show the end Date.

Reply:
Looks like this is an existing issue! I'll take a stab at fixing it. 

Do you think just setting the start and end date to null when the status report is generated is sufficient?